conf/app.src

```diff
@@ -0,0 +1,6 @@
+SOURCE_URL=https://github.com/duniter/cesium/releases/download/v1.3.11/cesium-v1.3.11-web.zip
+SOURCE_SUM=
```

Comment:
To get this value, download the file in SOURCE_URL, and run:

```
sha256sum cesium-v1.3.11-web.zip
```

And note the value. I have :

> $ sha256sum cesium-v1.3.11-web.zip 
fcac385f959178fc4ef8ac323b482c7d618bc48f67d6cc189229f91629b144be  cesium-v1.3.11-web.zip

So: 

```suggestion
SOURCE_SUM=fcac385f959178fc4ef8ac323b482c7d618bc48f67d6cc189229f91629b144be
```

Reply:
Thank you for explaining. I have an option in nautilus that can check if a checksum is correct but I didn't know how to obtain it. Easy! Cool!